src/controllers/GetPosts.ts

```diff
@@ -0,0 +1,58 @@
+import { Response, Request } from 'express'
+import { User, Post, Like, Tag, Animal } from '../db'
+import { Op } from 'sequelize'
+
+const getPosts = async (req: Request, res: Response) => {
+  const { tagId, animalId, searchCountent } = req.query
+  console.log(req.query)
+
+  let filterData = { }
+  tagId && animalId
+    ? filterData = {
+      content: {
+        [Op.substring]: searchCountent || ''
+      },
+      TagId: tagId,
+      AnimalId: animalId
+    }
+    : animalId
+      ? filterData = {
+        content: {
+          [Op.substring]: searchCountent || ''
+        },
+        AnimalId: animalId
+      }
+      : tagId
+        ? filterData = {
+          content: {
+            [Op.substring]: searchCountent || ''
+          },
+          TagId: tagId
+        }
+
+        : filterData = {
+          content: {
+            [Op.substring]: ''
+          }
+        }
```

Comment:
you are using nested ternary operators
this makes the code unreadable

use the `if`, `else if` and `else` 

and i want from you to add [this rule](https://eslint.org/docs/rules/no-nested-ternary) to `.eslintrc.json`

and you can look at [this article](https://medium.com/@benlmsc/stop-using-nested-ternary-operators-heres-why-53e7e078e65a)  to know more about this problem